simpleast-core/src/main/java/com/discord/simpleast/core/node/Node.kt

```diff
@@ -22,4 +20,18 @@ open class Node<R> {
   }
 
   open fun render(builder: SpannableStringBuilder, renderContext: R) {}
```

Comment:
I was tempted to change the render function here, but didn't know if that wouuld conflict with any downstream types.